doc/managing_wallet.md

```diff
@@ -0,0 +1,115 @@
+
+The `-rpcwallet` is the name of the wallet that will be encrypted.
+
+The term "encrypt the wallet" used here is not very accurate. This command only encrypts only the private key. All other wallet information, such as transactions, is still visible.
+
+Note that if the passphrase is lost, all the coins in the wallet will also be lost forever.
+
+### 1.3 Backing Up The Wallet
+
+Wallets can be safely copied to another destination. This backup file should be stored offline, such as on a USB drive, another computer, or an external hard drive.
+
+If the wallet and backup are lost for any reason, the bitcoins related to this wallet will become permanently inaccessible.
+
+The command is `bitcoin-cli backupwallet "destination"`. The destination parameter must include the name of the file. Otherwise, the command will return an error message like "Error: Wallet backup failed!" for descriptor wallets. If it is a legacy wallet, it will be copied and a file will be created with the default file name `wallet.dat`.
+
+`$ bitcoin-cli -rpcwallet="wallet-01" backupwallet /home/node01/Backups/backup01.dat`
+
+### 1.4 Backup Frequency
+
+The Bitcoin Core wallet was originally a collection of unrelated private keys with their associated addresses. If a non-HD wallet had generated a key/address, had given that address out and then had restored a backup from before that key's generation, then any funds sent to that address would have been lost definitively.
+
+However, [version 0.13](https://github.com/bitcoin/bitcoin/blob/master/doc/release-notes/release-notes-0.13.0.md) introduced HD wallets. Users no longer lose funds when restoring old backups, as long as the addresses used were from the wallet's HD seed (since all private keys can be derived from the seed).
+
+So, theoretically, a single backup is enough. But it is recommended to make regular backups (1 time a day or a week) or when there is a relevant amount of new transactions in the wallet.
+
+### 1.5 Restoring The Wallet From a Backup
+
+To restore a wallet, an empty one must be created  first.  The backup file then rewrites the `wallet.dat` of this new wallet.
+
+The user must unload the wallet and load it again after replacing the file.
+
+```
+$ bitcoin-cli createwallet "restored_wallet"
+$ bitcoin-cli unloadwallet "restored_wallet"
+$ cp ~/Backups/backup01.dat ~/.bitcoin/wallets/restored_wallet/wallet.dat
+$ bitcoin-cli loadwallet "restored_wallet"
+```
+
+After these steps, the wallet balance can be checked.
+
+`$ bitcoin-cli -rpcwallet="restored_wallet" getbalance`
+
+### 1.6 `-rescan` and `-reindex`
+
+The `-rescan` argument rescans the blockchain for missing wallet transactions during startup.
+
+`$ bitcoind -rescan`
+
+This is only necessary if there are missing transactions after restoring the wallet.
+
+A pruned node, however, is incompatible with the `-rescan` option, since it does not have the data to check for relevant transactions.
```

Comment:
```suggestion
A pruned node, however, is incompatible with the `-rescan` option since it does not have the data to check for relevant transactions.
```